packages/clay-navigation-bar/src/index.tsx

```diff
@@ -0,0 +1,150 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React, {useEffect} from 'react';
+
+import ClayButton from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+import ClayLink from '@clayui/link';
+
+import {removeCollapseHeight, setCollapseHeight} from './Helpers';
+import {useTransition} from './Hooks';
+
+interface Props extends React.HTMLAttributes<HTMLDivElement> {
+	/**
+	 * An array of items that will be rendered on the dropdown.
+	 */
+	items: Array<{active?: boolean; href?: string; label: string}>;
+
+	/**
+	 * Determines the style of the Navigation Bar
+	 */
+	inverted?: boolean;
+
+	/**
+	 * This property is used to pass a MouseEvent from a NavItem to a
+	 * function when this element is clicked.
+	 */
+	onItemClicked?: (
+		event: React.MouseEvent<HTMLLIElement, MouseEvent>
+	) => void | undefined;
+
+	/**
+	 * Path to the location of the spritemap resource.
+	 */
+	spritemap: string;
+}
+
+const ClayNavigationBar: React.FunctionComponent<Props> = ({
+	className,
+	inverted = false,
+	items,
+	onItemClicked,
+	spritemap,
+	...otherProps
+}) => {
+	const [visible, setVisible] = React.useState(false);
+	const contentRef = React.useRef<HTMLDivElement>(null);
+	const [
+		transitioning,
+		handleTransitionEnd,
+		handleClickToggler,
+	] = useTransition(visible, setVisible, contentRef);
+
+	return (
+		<nav
+			{...otherProps}
+			className={classNames(
+				className,
+				'navbar',
+				'navbar-collapse-absolute',
+				'navbar-expand-md',
+				'navbar-underline',
+				'navigation-bar',
+				{
+					'navigation-bar-light': !inverted,
+					'navigation-bar-secondary': inverted,
+				}
+			)}
+		>
+			<div className="container-fluid container-fluid-max-xl">
+				<ClayLink
+					className={classNames(
+						'navbar-toggler',
+						'navbar-toggler-link',
+						{
+							collapsed: !visible,
+						}
+					)}
+					displayType="secondary"
+					onClick={handleClickToggler}
+				>
+					{items.map(({active, label}) => {
```

Comment:
It might be best to use `items.find(item => item.active)`

Reply:
Makes sense :)